src/test/java/bio/terra/service/filedata/google/gcs/GcsPdaoTest.java

```diff
@@ -70,6 +78,86 @@ public void testGetBlobWithHash() {
     }
   }
 
+  @Test
+  public void testGetBlobContentsMatchingPath() {
+    String uuid = UUID.randomUUID().toString();
```

Comment:
This could be declared as `UUID` instead of `String` and the code would work the same.